config/default.yml

```diff
@@ -1989,6 +1989,13 @@ Naming/MethodName:
   SupportedStyles:
     - snake_case
     - camelCase
+  # Method names matching patterns are always allowed.
+  #
+  #   IgnoredMethodPatterns:
+  #     - '\A\s*onSelectionBulkChange\s*'
+  #     - '\A\s*onSelectionCleared\s*'
+  #
+  IgnoredMethodPatterns: []
```

Comment:
How do we typically name this? I want to make sure we are consistent with the name.

Reply:
Certainly, it can be made consistent with `IgnoredPatterns` using `IgnoredPattern` module. I first updated the option name added in this PR to `IgnoredPatterns`. 
After that I will open another PR to make the existing `IgnoredMethodPatterns` consistent with `IgnoredPatterns`.

Reply:
> After that I will open another PR to make the existing `IgnoredMethodPatterns` consistent with `IgnoredPatterns`.

I opened a PR #7313.